src/status_im/subs/wallet/wallet.cljs

```diff
@@ -24,6 +24,11 @@
                                                 (set))]
     (assoc account :network-preferences-names network-preferences-names)))
 
+(rf/reg-sub
+ :wallet/qr-code
+ :<- [:wallet]
+ :-> :qr-code)
```

Comment:
hmm, this :qr-code is saved when the scanning is done? 🤔 
maybe it's better we have a `generic-scanner` key in the db that saves these results and we read from there? I think conceptually the generic-scanner needs a clear plan of implementation otherwise we're going to have some logic for the scanner built close to it and others not.
Personally I would be in favour of keeping all the scanner code/events close to the generic scanner code and treat it like it's own set of functionality.

cc @alwx, @ulisesmac @cammellos  wdyt?

Reply:
It's not needed anymore. I removed it. @J-Son89 @cammellos @ulisesmac